src/plugins/scene3d/Scene3D.hh

```diff
@@ -18,6 +18,9 @@
 #ifndef IGNITION_GUI_PLUGINS_SCENE3D_HH_
 #define IGNITION_GUI_PLUGINS_SCENE3D_HH_
 
+#include <ignition/msgs/boolean.pb.h>
+#include <ignition/msgs/stringmsg.pb.h>
```

Comment:
Can these be removed?

Reply:
Yup, done a393787385e8b469bcaae8a5d44f8481717ed24c